tests/test_kms/test_kms_grants.py

```diff
@@ -137,23 +138,18 @@ def test_revoke_grant():
 
 
 @mock_kms
-def test_revoke_grant_by_token():
-
+def test_revoke_grant_raises_when_grant_does_not_exist():
```

Comment:
Why are you modifying/removing an existing test here?  Why not just add a new one to cover your changes?

Reply:
As mentioned in the PR description. The `test_revoke_grant_by_token` code was a 100% duplicate of `test_retire_grant_by_token`. It had the same exact code, so it wasn't testing revoking a grant by grant id. 

That would have been actually impossible, because in the [KMS API](https://docs.aws.amazon.com/kms/latest/APIReference/API_RevokeGrant.html) you cannot revoke a grant by token, you can only retire a grant by token.
As a matter of fact, boto3 [revoke_grant](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/kms.html#KMS.Client.revoke_grant) has no functionality for revoking a grant by passing a token id.
So the test `test_revoke_grant_by_token` was:
 1. inaccurate, at least in the naming
 2. a duplicate of `test_retire_grant_by_token`, as the code inside the test was 100% the same.

Hence my decision to remove it.